### PR TITLE
Details of VBAT usage

### PR DESCRIPTION
**Pull request by howels**

### CODE REVIEW

File: docs/manufacturer/connector-standard.mdx [hunk 1]
```diff
@@ -43,6 +43,13 @@ An additional 2-pin connector for power (ext. power) can be used for high-powere
 
 We recommend using twisted wires to eliminate any confusion about the mirroring of the connector and to ensure that the same wiring order is used on both sides of the connection.
 
+V+ connection from the ESC to FC will typically carry VBAT voltage direct from the battery connection. The ESC V+ connection is an Input Voltage to the FC whilst RX, GPS and other connectors' 
+V+ pads carry output from the FC's onboard voltage regulators. 
+
+In some cases VTX or camera connectors may offer VBAT voltage directly but due to voltage fluctuations induced by the motors the use of VBAT direct to VTXs or cameras is discouraged. 
+To minimise the risk to sensitive VTX hardware it is advisable to provide an additional high voltage regulator for such components.
```
Comment (haslinghuis) [inline docs/manufacturer/connector-standard.mdx:50]:
```suggestion
To minimize the risk to sensitive VTX hardware it is advisable to provide an additional high voltage regulator for such components. 
```
Looked it up:
image

Comment (howels) [reply docs/manufacturer/connector-standard.mdx:50]: Yes, I'm a Brit. Do we have a particular standard for this? I don't think any meaning is lost.

Comment (haslinghuis) [reply docs/manufacturer/connector-standard.mdx:50]: Counting 23 x `minimise` and 13 x `minimize` in docs folder :)

**Review by haslinghuis — APPROVED** [review]

File: docs/manufacturer/connector-standard.mdx (hunk 1 above)
Comment (howels) [reply docs/manufacturer/connector-standard.mdx:50]: Ok. Leaving it as "ise" which is also the Australian spelling

Comment (sugaarK) [reply docs/manufacturer/connector-standard.mdx:50]: Yea we speak kings English here not American

**Review by blckmn — APPROVED** [review]

**Pull request merged**